pkg/devfile/adapters/docker/component/adapter.go

```diff
@@ -39,6 +39,12 @@ type Adapter struct {
 	supervisordVolumeName     string
 }
 
+func (a Adapter) Build(parameters common.BuildParameters) (err error) { return nil }
+
+func (a Adapter) Deploy(parameters common.DeployParameters) (err error) { return nil }
+
+func (a Adapter) DeployDelete(manifest []byte) (err error) { return nil }
+
```

Comment:
Need to throw appropriate error message

Reply:
Did we determine we didn't need error messages here. @EnriqueL8 marked this as resolved without any changes being made to the code!!

Reply:
Might have miss clicked! Thanks for catching this steve! They removed the platform adapter from the main repo, so will need this.